context/app/static/js/components/home/HuBMAPDatasetsChart/HuBMAPDatasetsChart.tsx

```diff
@@ -93,6 +94,12 @@ function HuBMAPDatasetsChart() {
           assayTypeMap: datasetTypeMap,
         });
       },
+      getAriaLabel: (d) => {
+        const organ = String(d?.bar?.data?.organ);
+        const assay = d.key;
+        const count = d.key ? d?.bar?.data[d.key] : null;
+        return `${count} ${organ} datasets with assay type ${assay}.`;
```

Comment:
We should account for `null`'s and `undefined`'s string representations in template strings being `"null"/"undefined"` e.g.:
```js
const test = `${null} ${"Stomach"} datasets with assay type ${"scRNA-seq"}` 
console.log(test)
> "null Stomach datasets with assay type scRNA-seq"
const test2 = `${undefined} ${"Stomach"} datasets with assay type ${"scRNA-seq"}` 
console.log(test2)
> "undefined Stomach datasets with assay type scRNA-seq"
```

Maybe this could work?

```js
// We can use optional chaining with indexed objects like this:
const count = d?.bar?.data?.[d.key];
if (count) {
  return `${count} ${organ} datasets with assay type ${assay}.`;
}
return `${organ} datasets with assay type ${assay}.`
```